ivy/functional/frontends/torch/creation_ops.py

```diff
@@ -93,6 +93,55 @@ def empty_like(
     return ret
 
 
+@to_ivy_arrays_and_back
+def empty_strided(
+    size,
+    stride,
+    *,
+    dtype=None,
+    layout=None,
+    device=None,
+    requires_grad=False,
+    pin_memory=False,
+):
+    if dtype == "int8":
```

Comment:
If these are unsupported dtypes, you could make use of the [ivy decorator](https://github.com/asuzukosi/ivy/blob/d92979a49735bf2aa3741b28df8157fc69e1ce55/ivy/func_wrapper.py#L1632C9-L1632C9) for this use-case.

Reply:
Thanks for reviewing. I have removed the if elif statements as empty_strided functions supports all dtypes. Can you please review again?